Documentation/GettingStarted.md

```diff
@@ -557,28 +557,30 @@ This is how HTTP requests are wrapped in Rx. It's pretty much the same pattern l
 
 ```swift
 extension Reactive where Base: URLSession {
-    public func response(_ request: URLRequest) -> Observable<(Data, HTTPURLResponse)> {
+    public func response(request: URLRequest) -> Observable<(response: HTTPURLResponse, data: Data)> {
         return Observable.create { observer in
-            let task = self.dataTaskWithRequest(request) { (data, response, error) in
+
+            let task = self.base.dataTask(with: request) { (data, response, error) in
+            
```

Comment:
Can you please remove the two extra empty lines here? (or at least the one above)